docs/social/online-matches/setup.sql

```diff
@@ -0,0 +1,9 @@
+CREATE STREAM message_stream (
```

Comment:
Is this file leftover from other PR as well?

Reply:
Yup, that's gone.